.circleci/reference/VBFoffshell_69.lhe

```diff
@@ -42,7 +42,7 @@ Output from the JHUGenerator v7.5.0 described in arXiv:1001.3396 [hep-ph], arXiv
 -->
 <init>
 2212 2212   6500.0000000000000000    6500.0000000000000000 0 0  263000  263000  4  1
- 1.5365061E-03  2.8053372E-04  1.0000000E+00  69
+ 1.5346357E-03  2.8055356E-04  1.0000000E+00  69
```

Comment:
Is that a floating-point precision issue?

Reply:
This is due to a mistake in making a commit. I accidentally overwrote this file with the single offshell grid I make. This undoes it.

Reply:
Got it, thanks!